packages/head/index.tsx

```diff
@@ -24,12 +49,20 @@ export default function HashiHead(props: HashiHeadProps): React.ReactElement {
       <meta name="theme-color" content="#000" key="themeColor" />
       {whenString(
         props.description,
-        <meta
-          name="description"
-          property="og:description"
-          content={props.description}
-          key="description"
-        />
+        <>
+          <meta
+            name="description"
+            property="og:description"
+            content={props.description}
+            key="description"
+          />
+          <meta
+            // TODO: are we sure we need this? Seems like will fall back to OG
```

Comment:
@BRKalow wanted to confirm - do we strictly need `twitter:description`? It seems like it should fall back to `og:description` (ref: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup). Wondering whether we should omit it (or perhaps duplicate all such fallback tags with `twitter:` specific versions, for consistency?)

Reply:
I think you should be able to test this both with & without `twitter:description` using the [Twitter card validator](https://cards-dev.twitter.com/validator). Have you tried it out yet to help confirm the necessity of this tag?

Reply:
UGH that's so annoying 😭 but great tests! 🙌🏻 Glad that was fairly straightforward to test.